egui/src/containers/collapsing_header.rs

```diff
@@ -236,6 +238,28 @@ impl CollapsingHeader {
         self.show_background = show_background;
         self
     }
+
+    /// Use the provided function to render a different `CollapsingHeader` icon.
+    /// Defaults to a triangle that animates as the `CollapsingHeader` opens and closes.
+    ///
+    /// For example:
+    /// ```
+    /// 
+    /// # egui::__run_test_ui(|ui| {
+    /// fn circle_icon(ui: &mut egui::Ui, openness: f32, response: &egui::Response) {
+    ///     let stroke = ui.style().interact(&response).fg_stroke;
+    ///     ui.painter().circle_filled(response.rect.center(), 2.0 + openness, stroke.color);
```

Comment:
```suggestion
    ///     let radius = egui::lerp(2.0..=3.0, openness);
    ///     ui.painter().circle_filled(response.rect.center(), radius, stroke.color);
```

thanks for adding example code! Perhaps this would make it even more self-documenting!